### PR TITLE
Setup PostgreSQL

### PR DESCRIPTION
**Pull request by nurmerey**
PR  for issue #2
- [x] followed the instruction on https://tutorial-extensions.djangogirls.org/en/optional_postgresql_installation/

Note: to run and test this locally, you need to do the following parts of the above tutorial:
1. [Install PostgreSQL](https://www.dropbox.com/s/wi7vnlqf06tmffm/Screenshot%202019-12-23%2015.21.27.png?dl=0)
2. Skip sections "create database" and "update settings"
3. [Installing PostgreSQL package for Python](https://tutorial-extensions.djangogirls.org/en/optional_postgresql_installation/#installing-postgresql-package-for-python). If using Mac OS Mojave and have issue installing `pip install psycopg2`, then try `sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -target /`
4. run `python manage.py migrate`
5. You should see the following default Django tables in the DB (using the app Postico to see the db): 
Screenshot 2019-12-23 15 21 27

### CODE REVIEW

File: projectsplatform/settings.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 
 # SECURITY WARNING: don't run with debug turned on in production!
+SECRET_KEY = 'dulwepf_b#@eh19w)==v+pxvw$u$2gbxpwoevmzg)h-pl1mded'
```
Comment (nurmerey) [inline projectsplatform/settings.py:24]: @kellim was right, it doesn't run without the secret key.

Comment (natsteinmetz) [reply projectsplatform/settings.py:24]: Hmhm, ok. We should still not have have the secret_key on Github, but ideally in an env variable. Same for the database details below. 

Would you be comfortable setting that up, @nurmerey, or would you prefer I do that? 
Considering this is only for local development at this point, I'm fine keeping it for now such that you are able to move on. I'd create a story to switch the project over to use environment variables though, before we deploy it.

Let me know what you prefer, @nurmerey? 😃

**Review by natsteinmetz — APPROVED** [review]

File: projectsplatform/settings.py (hunk 1 above)
Comment (kellim) [reply projectsplatform/settings.py:24]: @natsteinmetz Would python-dotenv be acceptable for using in development? https://preslav.me/2019/01/09/dotenv-files-python/

Comment (nurmerey) [reply projectsplatform/settings.py:24]: @natsteinmetz I have created a separate story for the secret key and db details - #14

**Pull request merged**